source/install/cloud-changelog.md

```diff
@@ -4,13 +4,41 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2022-05-25](#release-2022-05-25)
 - [Release 2022-05-12](#release-2022-05-12)
 - [Release 2022-04-28](#release-2022-04-28)
 - [Release 2022-04-13](#release-2022-04-13)
 - [Release 2022-03-30](#release-2022-03-30)
 - [Release 2022-03-16](#release-2022-03-16)
 - [Release 2022-03-08](#release-2022-03-08)
 
+## Release 2022-05-25
+
+### Improvements
+
+#### User Interface (UI)
+ - To keep users in Mattermost when opening documentation links from the **System Console > Plugin** settings page, all the links now open in another tab.
+ - Improved right-hand side thread reply input and scrolling behavior by making the reply to thread input sticky.
+ - Users are no longer hidden from search results in the "Add members" modal, even if they are already members of the channel.
+ - Applied new designs for the Login screen:
+     - Default login
+     - OAuth options
+     - Custom branding
+     - MFA token
+ - Changed **Actions** post menu hover text to **Message Actions**.
+
+#### Administration
+ - Enabled ``always-on`` and ``default-on`` for Collapsed Reply Threads system configuration.
```

Comment:
What's the impact of these options? Are collapsed reply threads now always on?

Reply:
Clarified the text.